pkg/api/globalquery/client/stream.go

```diff
@@ -92,6 +131,15 @@ func (sse *SSEClient) Query(ctx context.Context, args *query.Args) (*results.Res
 	}
 	defer resp.Body.Close()
 
+	// Handle RFC 9457
+	if strings.EqualFold(resp.Header.Get("Content-Type"), "application/problem+json") {
```

Comment:
Handling for Huma responses (since this uses a default `http.Client` instead of `httpc` this was missing).

Reply:
Nit: could be a function in itself.

Reply:
Will be solved as part of https://github.com/fako1024/httpc/issues/42 instead.